var/spack/repos/builtin/packages/parallel-netcdf/package.py

```diff
@@ -25,8 +25,7 @@ def url_for_version(self, version):
 
         return url.format(version.dotted)
 
-    version('develop', branch='develop')
-    version('master', branch='master')
+    version('develop', branch='master')
```

Comment:
Why these changes?

Reply:
There is no `develop` branch in the repository actually. Only `master` and, as far as I remember, version `develop` has a special meaning in Spack and `master` does not. So, I made branch `master` the `develop` version.

Reply:
See #1983, both master and develop have special handling, so you can call this master